src/Playwright.CLI/Playwright.CLI.csproj

```
@@ -21,7 +21,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <AssemblyVersion>1.0.1</AssemblyVersion>
```

This needs a version bump, or it will get rejected.
```suggestion
    <AssemblyVersion>1.0.2</AssemblyVersion>
```